### ISSUE TITLE
Performance of Fast-AdvProp

### ISSUE THREAD

**Issue by Equationliu** [meijieru/fast_advprop#1]
By running
```
bash scripts/run_fastadvprop.sh
```
I got only 76.1% Top-1 accuracy, which is even lower than the performance of vanilla training as reported in the paper.

Also, note that the vanilla  top-1 accuracy of ResNet-50 in repo [pytorch-classification-advprop](https://github.com/tingxueronghua/pytorch-classification-advprop) reaches 76.67%.
